app/models/revision_content.rb

```diff
@@ -19,6 +19,14 @@ def all_content
     end
   end
 
+  def keys_in_order
+    [:name, :content_body] + node.all_content.keys.reject do |key|
```

Comment:
Would it be worth us including `summary` and `short_summary` as well? They are now fields which are also a part of the base Node class.